crates/core/src/dht/actions/remove_queued_holding_workflow.rs

```diff
@@ -6,16 +6,31 @@ use crate::{
 };
 use futures::{future::Future, task::Poll};
 use snowflake::ProcessUniqueId;
-use std::{pin::Pin, sync::Arc};
+use std::{pin::Pin, sync::Arc, time::Duration};
+
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+pub enum HoldingWorkflowQueueing {
```

Comment:
Comments on enums and functions would be tremendously helpful in understanding this stuff and conveying intent. I know most stuff is undocumented, but at least documenting new stuff would help